### ISSUE TITLE
Time warp

### ISSUE THREAD

**Issue by martin-ez** [martin-ez/project-titan#16]
Could be useful for play testing and it can be a game feature itself.

**Comment by martin-ez** [comment]
From the 0.16-0.19 assessment in #40, and the answer is that the lever already existed rather than that a release added one: `Time<Virtual>::set_relative_speed`, which `Time<Fixed>` accumulates from. So warping time runs more fixed ticks per frame rather than making each tick longer, which is what keeps invariant 2 intact — the simulation still sees whole ticks of the same length, and the same inputs give the same traffic at any warp factor. That is the property this issue has to preserve, and it comes for free from doing it this way rather than by scaling anything in the systems themselves.

One caution from 0.19 to go with it: `commands.delayed().secs(1.0)` schedules against elapsed time, not ticks. Anything a rover can observe must not be scheduled that way, or a jam stops being reproducible.

**Comment by martin-ez** [comment]
Moved out of #27 and into the core loop, and given #66 as its blocker.

It was parked behind the whole simulation chain, so it would have arrived after every system it makes easier to test. Its own body already said why that was wrong — "could be useful for play testing" — and invariant 5 agrees: whatever it takes to watch a system run while it runs is part of building it, not a reward for having finished.

The mechanism is settled by #66 rather than by this issue. Fast-forward is more ticks per unit of real time, not longer ticks: scaling `Time<Fixed>`'s timestep would change how far a rover moves in one tick and make 8x diverge from 1x. `Time<Virtual>`'s relative speed is what this drives. What is left here is the player-facing part — the control, the speeds offered, and what the interface shows about which one is running.

#65 is what keeps the camera out of it, so fast-forwarding the world does not fast-forward the view.

**Comment by martin-ez** [comment]
Claimed by `feat/16-time-warp` via `scripts/track.sh claim`.

**Comment by martin-ez** [comment]
Built by `feat/16-time-warp`, waiting on a human merge of #88.
